aws/policy/security-services.yaml

```diff
@@ -28,6 +28,8 @@ Statement:
           - arn:aws:iam::aws:policy/IAMReadOnlyAccess
           - arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM
           - arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole
+          - arn:aws:iam::aws:policy/AmazonEKSClusterPolicy
```

Comment:
In general this looks great. It would be good to get @mattclay's opinion on allowing this policy to be attached to roles. Here's an example where the same underlying issue this policy exposes has been deemed not to be a viable option- https://github.com/ansible/ansible-core-ci/pull/197#discussion_r261330886

An alternative would be to create a static role for the CI account, and only try to create the role/attach the policy if it does not already exist. Then users could still have the benefit of not having to create the role for their account ahead of time, but the CI account would not need this permission.

Reply:
I agree that the static role is our only option currently, until we're able to run the terminator in all regions.

Reply:
Works for me. AFAICT that's not something I can do through this repo though, unless I'm missing some breadcrumb the CreateSecret example should lead me too.  How do I create a static role?

Reply:
@s-hertel @mattclay can either of you point me to how I create the static role?  I'm pretty sure I don't have privs to do so in the console as I can't see permissions for any existing roles and none of the roles I can see look like they'd cover the noted example.

Reply:
Sorry, @jillr. I already created the role when we talked about it in Slack last week. The role name is test-eks-role (to keep the naming pattern consistent), in case you forgot.